src/config.rs

```diff
@@ -49,6 +49,24 @@ pub struct PluginConfiguration {
     #[serde(with = "serde_regex")]
     pub exclude_urls: Vec<Regex>,
 
+    // Header forwarding settings
+    /// Forward the access token in a header
+    pub forward_access_token: bool,
+    /// The header name that will be used for the access token
+    /// If the header name is empty, the access token will be forwarded in the authorization header
+    pub access_token_header_name: String,
```

Comment:
Why not make this an `Option` and delete the `forward_access_token` parameter entirely? Then we don't set a header if `access_token_header_name` is omitted from the config.